action/protocol/execution/evm/evm.go

```diff
@@ -199,21 +197,22 @@ func getChainConfig() *params.ChainConfig {
 	return &chainConfig
 }
 
-func executeInEVM(evmParams *Params, stateDB *StateDBAdapter, gasLimit uint64) ([]byte, uint64, uint64, string, bool, error) {
+//Error in executeInEVM is a consensus issue
+func executeInEVM(evmParams *Params, stateDB *StateDBAdapter, gasLimit uint64, isPostBering bool) ([]byte, uint64, uint64, string, uint64, error) {
```

Comment:
do not pass isPostBering as input, this won't scale if we have more height change in the future.
You can use stateDB.hu.IsPostBering() inside executeInEVM(), see comment below